src/overrides/safari_version.py

```diff
@@ -0,0 +1,44 @@
+import os
+import subprocess
+import plistlib
+
+
+_safari_version = ""
+_SAFARI_BUNDLE = "com.apple.Safari"
+
+
+def get() -> str:
+    global _safari_version
+    if _safari_version != "":
+        return _safari_version
```

Comment:
I don't think we gain much from having this global here. As far as I can tell the only time this would be useful is if you make a change to safari.py and that file loads but not this one. I would just remove it.